frontend/src/lib/components/HabitsList.svelte

```diff
@@ -3,16 +3,38 @@
     export let time;
     export let iconCount;
     export let habitId;
- 
-    let menuOpen = false;
+    export let menuOpen;
+    export let storedId;
+   
+    function handleOpening(id){
+        let dropdown = document.getElementById(`dropdown${id}`);
+
+        if (menuOpen == false) {
+        // Displays menu
+        dropdown.style.display = "block";
+        menuOpen = true;
+        storedId = id;
+        } else if(storedId != id) {
+            // Closes previous dropdown and opens the new one
+            document.getElementById(`dropdown${storedId}`).style.display = "none";
+            dropdown.style.display = "block";
+            storedId = id;
+            console.log('3rd con');
+        } else {
+            // Closes the dropdown
+            dropdown.style.display = "none";
+            menuOpen = false;
+            storedId = id;
+        }
+    }
 </script>
- 
+  <!--Clickable-->
 <div class="bg-white rounded-xl shadow-lg p-1 flex justify-between">
-    <a href={`habits/${habitId}`} class="flex">
+    <a href={`habits/${habitId}`} class="flex w-full">
         <div>
             <!-- svelte-ignore a11y-missing-attribute -->
```

Comment:
Don't ignore the accessibility warnings. They are there for a reason. 

https://svelte.dev/docs/accessibility-warnings